wix/main.wxs

```diff
@@ -168,10 +168,10 @@
           The product icon is the graphic that appears in the Add/Remove
           Programs control panel for the application.
         -->
-        <Icon Id='ProductICO' SourceFile='wix\notion.ico'/>
+        <Icon Id='ProductICO' SourceFile='wix\volta.ico'/>
```

Comment:
You can find a new (not amazingly awesome, but perhaps serviceable for now?) icon in volta-cli/assets.

Reply:
Perfect, will add it!

Reply:
Added it in 6c69bf03. Thank you!